chapter-08/end/Engine/Particles/System.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace chapter_08.Engine.Particles
+{
+    class System
```

Comment:
Purpose of this class?

Reply:
Most tutorials have a particle system class that contains more than 1 emitter. I guess if I had multiple torches on a level, I'd need one good way to manage all of them at the same time. However this class isn't needed here, right now, so I removed it.